build.sbt

```diff
@@ -17,6 +17,7 @@ val commonDeps = Seq(
   "com.rojoma"        %% "simple-arm"          % "[1.2.0,2.0.0)",
   "com.rojoma"        %% "simple-arm-v2"       % "2.0.0" % "optional",
   "com.rojoma"        %% "rojoma-json-v3-grisu"  % "1.0.0" % "optional",
+  "org.scalatra"      %% "scalatra"            % "2.2.2" % "optional",
```

Comment:
Note:  we really need to change `optional` to provided, for all the reps.  `optional` will result in Scalatra being pulled into every sbt assembly for every project that pulls in thirdparty-utils, which is not what we want.

Robert has some disagreements about this, but this is just how sbt-assembly currently works.   I would say it's not worth the time to fix it in SBT / assembly, and just give up ideology and agree on being practical.